app/models/school.rb

```diff
@@ -47,6 +47,10 @@ def verify!
     update!(verified_at: Time.zone.now)
   end
 
+  def reject
+    update(rejected_at: Time.zone.now)
+  end
+
```

Comment:
Is there any reason you've made `School#reject` a non-bang method calling `School#update` whereas `School#verify!` is a bang method calling `School#update!`?

Reply:
I've added the following to the commit note. Do you think this is clear enough?

> To complement `School#verify!`. Note that this doesn't need to be a bang method because the return value of calling `update` is passed through `SchoolVerificationService#reject` and back to `Admin::SchoolsController#reject`. The `School#verify!` method needs to raise on error so that it triggers a transaction rollback in `SchoolVerificationService#verify`.